src/cls/rbd/cls_rbd_client.cc

```diff
@@ -1403,7 +1404,7 @@ void object_map_snap_add(librados::ObjectWriteOperation *rados_op)
 void object_map_snap_remove(librados::ObjectWriteOperation *rados_op,
                             const ceph::BitVector<2> &object_map)
 {
-  ceph::BitVector<2> object_map_copy(object_map);
+  ceph::BitVector<2> object_map_copy;//(object_map);
```

Comment:
@dillaman  This looks like it leaked to the commit unintentionally ? It causes `TestClsRbd.object_map_snap_remove` failure.

Reply:
Sorry, that was a left-over from trying to decode a very long compiler error deep within the copy constructor for ```std::vector```